ansible/create_template_box.yml

```diff
@@ -18,6 +18,7 @@
     - name: git_branch
       prompt: "git branch (e.g. master)"
```

Comment:
I'd change the prompts to a more standard convetion indicating this is a default value: `git branch [master]`

Reply:
but that is how it works, if you execute the script, the prompt will be:
`git branch (e.g. master) [master]`

Reply:
Ah, in that case, we could remove the `(e.g. master)`? :)